static/app/components/eventsTable/eventsTable.spec.jsx

```diff
@@ -2,6 +2,10 @@ import {render} from 'sentry-test/reactTestingLibrary';
 
 import EventsTable from 'sentry/components/eventsTable/eventsTable';
 
+jest.mock('sentry/utils/useRoutes', () => ({
+  useRoutes: jest.fn(() => []),
+}));
```

Comment:
we should have assertions that cover the new column to make sure that the urls are correct and deeplink into the correct point in time.

Reply:
Going to see if I can't find and update the mocked react-router link component to support objects passed to `to` in tests. Currently if you use an object on the link it renders an empty string